rusk-abi/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rusk-abi"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
Ditto

Reply:
No version change here for the time being, since we're going to merge into `rusk-abi-0.3`